rspace-bench/src/test/scala/coop/rchain/rspace/KryoRoundTripTest.scala

```diff
@@ -23,7 +23,7 @@ class KryoRoundTripTest extends FlatSpec with PropertyChecks with Matchers {
     assert(out == in)
   }
 
-  def roundTripSerialization[A: Arbitrary](
+  def roundTripSerialization[A: Arbitrary :Shrink](
```

Comment:
I expected that `Arbitrary` implies `Shrink`.

Reply:
It does not, at least not in scalacheck. I think it does in Haskell's quickCheck (shrink is a method of Arbitrary)